tangelo/toolboxes/ansatz_generator/uccsd.py

```diff
@@ -100,7 +101,7 @@ def __init__(self, molecule, mapping="JW", up_then_down=False, spin=None, refere
 
         # Default initial parameters for initialization
         # TODO: support for openshell MP2 initialization
-        self.var_params_default = "mp2" if (self.spin == 0 and not self.molecule.uhf) else "ones"
+        self.var_params_default = "mp2" if "mp2" in self.supported_initial_var_params else "ones"
```

Comment:
is that todo on L103 still relevant as a result of these changes?

Reply:
Yes. I do have it implemented somewhere as it is in the code for the UHF-FNO-UCCSD code that I implemented but was waiting for the QAI funding along with AGC. Could be added quickly if desired though.